src/scss/_theme_colors_default.scss

```diff
@@ -12,6 +12,12 @@ $emc-theme-feature-colors: (
 
         body-text: emc-black,
 
+        link: emc-link,
+        link-visited: emc-link-visited,
+        link-hover: emc-link-hover,
+        link-active: emc-link-active,
+        link-focus: emc-link-focus,
```

Comment:
Change these references to point to one of the existing colors in the palette instead of defining new colors.